src/classes/CW_AdminController.cls

```diff
@@ -0,0 +1,168 @@
+public with sharing class CW_AdminController {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.